pkg/arvo/sur/permission-store.hoon

```diff
@@ -0,0 +1,25 @@
+|%
++$  kind  ?(%black %white)
+::
++$  permission
+  $:  =kind
+      who=(set ship)
+  ==
+::
++$  permission-map  (map path permission)
+::
++$  permission-update
+  $%  [%create =path =permission]
+      [%delete =path]
+      [%add =path who=(set ship)]
+      [%remove =path who=(set ship)]
+  ==
+::
++$  permission-action
```

Comment:
A comment describing the difference between `%add`, `%remove`, `%allow`, and `%deny` could be useful.  Seems to be "`%add` (`%remove`) adds (removes) to the set of specially-handled ships; `%allow` (`%deny`) is same as `%add` (`%remove`) if `%white` else same as `%remove` (`%add`).

Reply:
Yes, your understanding is right. Will add comments